extensions-core/datasketches/src/main/java/org/apache/druid/query/aggregation/datasketches/hll/sql/HllSketchEstimateOperatorConversion.java

```diff
@@ -43,7 +43,7 @@
 
 public class HllSketchEstimateOperatorConversion implements SqlOperatorConversion
 {
-  private static final String FUNCTION_NAME = "HLL_SKETCH_ESTIMATE";
+  private static final String FUNCTION_NAME = "hll_sketch_estimate";
```

Comment:
it isn't case sensitive here, [we lowercase them all prior to adding to the map](https://github.com/apache/druid/blob/master/processing/src/main/java/org/apache/druid/math/expr/ExprMacroTable.java#L56), however we do not lower-case them while looking them up when parsing the expression, https://github.com/apache/druid/blob/master/processing/src/main/java/org/apache/druid/math/expr/ExprListenerImpl.java#L327 for macros, though this might technically be a bug because the built-in functions we do attempt to lowercase them first when looking up https://github.com/apache/druid/blob/master/processing/src/main/java/org/apache/druid/math/expr/Parser.java#L90

Reply:
btw, this can just extend `DirectOperatorConversion` and lowercase the function name (or not if we modify it to not be case sensitive for macros, though we should be thoughtful before making such a change)